content/articles/managing-docker-containers-using-command-line-interface/index.md

```diff
@@ -0,0 +1,102 @@
+
+```bash
+sudo docker volume ls -f dangling=true
+```
+
+> To avoid leaving dangling volumes while deleting Docker volumes, use the `-v flag`.
+
+```bash
+sudo docker rm -v <name-of-the-container>
+```
+
+To list the docker containers that are now executing, use the following sequence of instructions. Stop the specific container and remove it with the `-v` flag to prevent it from leaving dangling volumes behind.
+
+>One by one, run the commands below:
+
+```bash
+sudo docker container ls
+sudo docker stop my-container-01
+sudo docker rm -v my-container-01
+```
+The first command shows a list of all Docker containers that are currently running. After viewing the names of all the docker containers, stop a specific container using the second command.
+`sudo docker stop my-container-01`. write the name of the container after the word stop.
+
+### Removing docker containers and images
+
+Docker containers are not automatically deleted when they are stopped unless they are started using the `—rm` parameter. But, before deleting a docker container, you must first ensure that it is not operating. Containers can be stopped and deleted with the following instructions:
+
+```bash
+sudo docker ps -a
+sudo docker stop <name-of-the-container>
+sudo docker rm <name-of-the-container> 
+```
+
+The first command lists all of your system's containers. The container's state may be found in the status column. Stopping the container before removing it is required if it has not been exited. You must know the container ID of the container you want to delete before you delete them. The abbreviation `rm` represents remove.
+
+Remove all the containers linked with a docker image before removing it using the command below:
+
+```bash
+sudo docker rmi <The-ID-of-the-image>
+```
+From the above command, the `sudo` permits users to run programs with another user's security rights, `docker` shows that we are executing the command to a docker. `rmi` means remove the docker objects specified by the name given.
+
+### Using aliases
+
+There are times when we must repeat the same command-line instruction, which is made more difficult if the command in question contains parameters, and much more so if we must change the output to make it legible. Most of us end up making some sort of cheat sheet to keep track of all those complicated commands in case we need to execute them again.
+
+We may save time and effort by utilizing an `alias` command to avoid typing or duplicating the same command over and over again. Aliases allow us to execute a command or a group of instructions using a pre-defined `string` that we can tailor to our liking.
+
+To handle aliases, simply create and use aliases in your `/.bashrc` file and mention them.
+
+```bash
+alias dockrm='docker rm'
+alias docklist='docker ps -a'
+```

```

Comment:
Also, briefly explain these few lines of code. What does `ps` and `-a` represent? Etcetra.